templates/writing-task-topics.rst

```diff
@@ -0,0 +1,240 @@
+:orphan: true
+	 
+.. Based on: https://dmtn-030.lsst.io/v/DM-7096/index.html#task-topic-type, with learnings from the 4 sfp pages built in branch DM-8717
+
+###################
+Writing Task Topics
+###################
+
+This page describes how to document science pipelines tasks for the
+`pipelines.lsst.io <http://pipelines.lsst.io>`_ documentation project.
```

Comment:
That's curious - i assume i would've cut and pasted this in, and it defaults to the secure URL (in fact, if you either click on it, or try to insert the http form into the URL window, they will both revert back to the https site), so not sure how it crept in, but i corrected it, anyway.